src/corelibs/U2View/src/ov_phyltree/TreeViewer.cpp

```diff
@@ -209,6 +210,8 @@ void TreeViewer::createActions() {
 
     // Print Tree
     printAction = new QAction(QIcon(":/core/images/printer.png"), tr("Print Tree..."), ui);
+    printAction->setDisabled(QPrinterInfo::availablePrinters().isEmpty());
```

Comment:
Usually when there are no active printers I still can print to PDF file. How this case is handled?

Reply:
If there are no printers installed (even virtual pdf printer) qt will not show "Print" dialog.
I've checked it on my PC without printers installed.

Reply:
Here how it works on Linux when no printers installed: there is still "Save to PDF" option.

![image](https://user-images.githubusercontent.com/17791039/181061231-88c09e93-ba9c-42f7-97bb-053c9431d2d0.png)